README.md

```diff
@@ -0,0 +1,20 @@
+# Quilt Kotlin Libraries
+
+#### Note: At the moment, [Quilt Standard Libraries](https://github.com/QuiltMC/quilt-standard-libraries), and as such Quilt Kotlin Libraries, are not read for general use. Please make an issue or talk to the QKL team on discord before writing any PRs.
```

Comment:
This is a bit confusing to read

Reply:
Also `read` should probably be `ready`

Reply:
I'm aware :>

Reply:
Since when was QSL not ready for general use?

Reply:
The QSL repo says that itself, so since when? when ever that was written 😛